package.json

```diff
@@ -37,15 +38,20 @@
     "d3-collection": "^1.0.2",
     "d3-selection": "^1.0.4",
     "datalib": "^1.7.3",
```

Comment:
We still need `d3-collection` and `d3-selection` because of d3 map, but I plan to remove that to use javascript map instead